src/CommonDropdown.php

```diff
@@ -743,7 +743,7 @@ public function importExternal(
     ) {
 
         $value = trim($value);
-        if (strlen($value) == 0) {
+        if (empty($value)) {
```

Comment:
This change seems dangerous to me.
Is it possible to filter `0` values on inventory side ?

Reply:
Why should that be dangerous?

Reply:
CommonDropdown::import() that is finally called use empty on provided name, I really do not undesratnd why importExternal should only check empty string